interface/patient_file/history/history_full.php

```diff
@@ -254,18 +244,14 @@ function sel_related(e) {
 <form action="history_save.php" id="HIS" name='history_form' method='post' onsubmit="submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,'HIS',constraints)">
     <input type='hidden' name='mode' value='save'>
 
-    <div>
-        <span class="title"><?php echo htmlspecialchars(xl('Patient History / Lifestyle'),ENT_NOQUOTES); ?></span>
+    <div class="page-header">
+        <h1><?php echo htmlspecialchars(getPatientName($pid), ENT_NOQUOTES);?>&nbsp;<small><?php echo htmlspecialchars(xl('History & Lifestyle'),ENT_NOQUOTES); ?></h1>
     </div>
-    <div id='namecontainer_fhistory' class='namecontainer_fhistory' style='float:left;margin-right:10px'>
-  <?php echo htmlspecialchars(xl('for'),ENT_NOQUOTES);?>&nbsp;<span class="title"><a href="../summary/demographics.php" onclick='top.restoreSession()'><?php echo htmlspecialchars(getPatientName($pid),ENT_NOQUOTES); ?></a></span>
-    </div>
-    <div>
-        <input class="css_btn"  type="submit" value="<?php xl('Save','e'); ?>">
-
-        <a href="history.php" class="css_button" onclick="top.restoreSession()">
-            <span><?php echo htmlspecialchars(xl('Back To View'),ENT_NOQUOTES); ?></span>
+    <div class="btn-group">
+        <a href="history.php" class="btn btn-default" onclick="top.restoreSession()">
+            <i class="fa fa-chevron-left"></i>&nbsp;&nbsp;<?php echo htmlspecialchars(xl('Back To View'),ENT_NOQUOTES); ?>
```

Comment:
as above, could use the btn-transmit class rather than the chevron-left
(note if you look closely it appears the current strategy gives slightly different heights to each button)